Agreements/Internal/TCSE_17.txt

```diff
@@ -0,0 +1,6 @@
+COPX:EGSS:*:LYD:*:*:London TC SE:London S17:19000:*:^LYD
+COPX:EGSC:*:LYD:*:*:London TC SE:London S17:19000:*:^LYD
+COPX:EGLC:*:LYD:*:*:London TC SE:London S17:19000:*:^LYD
+COPX:EGKB:*:LYD:*:*:London TC SE:London S17:19000:*:^LYD
+COPX:EGTO:*:LYD:*:*:London TC SE:London S17:19000:*:^LYD
+COPX:EGMC:*:LYD:*:*:London TC SE:London S17:19000:*:^LYD
```

Comment:
Will be changing this agreement from the 22nd to match the r/w (RAD) next cycle:

TC SE to Sector **18** - 190 lvl by the S17/S18 boundary

I suggest using WAFFU as the FIX, perhaps?

Reply:
Fixed